group-coordinator/src/main/resources/common/message/StreamsGroupCurrentMemberAssignmentKey.json

```diff
@@ -0,0 +1,28 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The streams rebalance protocol is in development. This schema is subject to non-backwards-compatible changes.
+{
+  "type": "data",
+  "name": "StreamsGroupCurrentMemberAssignmentKey",
+  "validVersions": "14",
```

Comment:
Why is this 14?

Reply:
It makes sure that key types can be distinguished. If you look careful, you will see that every key schema for the offset commit topic has a unique valid versions field.

Reply:
I noticed that. So it is a way how to distinguish the keys on a byte level, because we do not have a unique identifier like the API key?

Reply:
Exactly that. It's not pretty, but it works